marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/FlowMvcTest.java

```diff
@@ -22,12 +21,16 @@
     FlowController flowController;
 
     private final static String PATH = "/api/flows";
-    private ObjectMapper mapper = new ObjectMapper();
 
     @Test
     void runIngestionStep() throws Exception {
         installReferenceModelProject();
-        loginAsTestUserWithRoles("hub-central-step-runner");
+        if(isVersionCompatibleWith520Roles()) {
+            loginAsTestUserWithRoles("hub-central-step-runner");
+        }
+        else {
+            loginAsTestUserWithRoles("hub-central-step-runner", "flow-operator-role");
```

Comment:
The test fails in 9.0-12 when trying to update job doc unless user has "flow-operator-role"

Reply:
No need to change, but this is a useful comment, and useful comments are better off in the code than in a PR comment. But I think the code is self-documenting enough.